utils/s2n_ensure.h

```diff
@@ -148,6 +149,7 @@ void *s2n_ensure_memmove_trace(void *to, const void *from, size_t size);
     #define CONTRACT_ASSIGNS(...)
     #define CONTRACT_ASSIGNS_ERR(...)
     #define CONTRACT_ASSUME(...)
+    #define CONTRACT_DECREASES(...)
```

Comment:
It looks like the [sidetrail tests are failing to build](https://us-west-2.codebuild.aws.amazon.com/project/eyJlbmNyeXB0ZWREYXRhIjoidUpNb2QzNjZzQ3lrWm1UQ2xzdFZMN1ZGYWE5dVlSQ1ZKd1BnZzZ6aXN1dlZXTmFuZWdvNnR0SitPaFlIcUdjaDZ6RU9KemY2Mi9ybXVBajF0RXo4NmZ1Rnl4dkZrT241UUY4MTVQMkRuNTQ9IiwiaXZQYXJhbWV0ZXJTcGVjIjoiemFRUDgvZUMreVhER0RKaiIsIm1hdGVyaWFsU2V0U2VyaWFsIjoxfQ%3D%3D/build/a59c4d0b-8bf7-4ba9-9819-502524c37c30) with these changes. It looks like `s2n_ensure.h` might need to be included in the [sidetrail harnesses](https://github.com/aws/s2n-tls/blob/main/tests/sidetrail/working/s2n-record-read-cbc-negative-test/s2n_record_read_wrapper.c)?